app/wol.py

```diff
@@ -1,17 +1,21 @@
 from flask import Flask, request, render_template, redirect, url_for, jsonify
 from flask_login import LoginManager, UserMixin, login_user, logout_user, login_required, current_user
+from scapy.all import Ether, sendp
 import socket
 import struct
 import subprocess
 import os
 import ipaddress
 import re
+import binascii
 
 port = os.environ.get('PORT', 5000)
 ping_timeout = os.environ.get('PING_TIMEOUT', 300)
 arp_timeout = os.environ.get('ARP_TIMEOUT', 300)
 tcp_timeout = os.environ.get('TCP_TIMEOUT', 1)
 arp_interface = os.environ.get('ARP_INTERFACE')
+l2_wol_paket = os.environ.get('L2_WOL_PAKET')
```

Comment:
```suggestion
l2_wol_packet = os.environ.get('ENABLE_L2_WOL_PACKET', 'false').lower() == 'true'
```
os.environ.get() always returns a string -> we need a boolean for the if statement. Default value should be existing WOL